### PR TITLE
ux: use consistent icons for sytem and template

### PR DESCRIPTION
**Pull request by aizatto**
## Hey, I just made a Pull Request!

Noticed that there were inconsistencies in the icons:

1. `plugins/catalog/src/apis/EntityPresentationApi/defaults.tsx`:
    1. system: @material-ui/icons/Business
    2. template: @material-ui/icons/LocationOn
3. `packages/app-defaults/src/defaults/icons.tsx`:
    1. system: @material-ui/icons/Category
    2. template: @material-ui/icons/FeaturedPlayList

Standardize on the `app-defaults`

System
Before:
image

After:
image

Template

Before:
image

After:
image

#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [x] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

**Review by awanlin — CHANGES_REQUESTED** [review]
Thanks for this @aizatto, agree that this isn't consistent. As the values in `@backstage/app-defaults` predates the `EntityPresentationApi` those should be seen as the proper default values, not the other way around. Do you mind making that adjustment, please? Change would be here: https://github.com/backstage/backstage/blob/1f6040e4bd110840e43043eb0dac8b48b686cdaa/plugins/catalog/src/apis/EntityPresentationApi/defaults.tsx#L38

Also, your before and after images don't show the Template changes. Could you update those?

Finally, we should also fix the fact that the Launch Template button also uses an icon for Templates that is different then what is used elsewhere: 
image

Code is here: https://github.com/backstage/backstage/blob/1f6040e4bd110840e43043eb0dac8b48b686cdaa/plugins/catalog/src/components/AboutCard/AboutCard.tsx#L169

**Comment by aizatto** [comment]
done as requested, uploaded new screenshots in place of original comment

File: .changeset/loud-walls-compare.md
Comment (alde) [inline .changeset/loud-walls-compare.md:2]: is this the correct plugin? looks like it's catalog that's changed

Comment (aizatto) [reply .changeset/loud-walls-compare.md:2]: sorry you're correct, this should be changed after i updated as per request

**Review by alde — APPROVED** [review]

File: plugins/catalog/src/apis/EntityPresentationApi/defaults.tsx
Comment (awanlin) [inline plugins/catalog/src/apis/EntityPresentationApi/defaults.tsx:47]: While this is correct as I requested I don't think I ever noticed that this was the actual default for Templates. I'd like to see what the Maintainers think but personally using `AddCircleOutline` makes more sense as that's what is used in the sidebar:

image

That seems more likely to be what someone associates with Templates, again my personal opinion here ;)

**Review by awanlin — APPROVED** [review]
Thanks for making the adjustments @aizatto, left one comment but it's an opinion and I do think this PR is technically correct so going to approve 🚀 

We still need a review/approval from the maintainers now either way

**Comment by aizatto** [comment]
thanks, will wait for approval

**Comment by aizatto** [comment]
which are the maintainers I should get approval from?

**Comment by freben** [comment]
I'm back next week.

Hmm. So this makes "this is a template" (a specific thing) and "I'd like to create something" (a generic action) the same icon? Not sure about that one. @benjdlambert / @backstage/scaffolder-maintainers ?

**Comment by aizatto** [comment]
@freben checking in :)

**Comment by benjdlambert** [comment]
@freben @aizatto yeah to be honest I'm fine with the systems Icon changes, but I think that the template icon changes are maybe unnecessary. They're indicative of two different use cases - so probably don't want the same icon here?

What do you think?

**Comment by freben** [comment]
Yep that was my thought.

**Comment by aizatto** [comment]
i've reverted changes to `AboutCard.tsx`

**Review by freben — APPROVED** [review]

**Comment by awanlin** [comment]
Awesome work @aizatto, thanks for your patience on this 🚀 

Thanks for the reviews @freben and @benjdlambert 👍

**Pull request merged**
